src/Shell/Task/CellTask.php

```diff
@@ -74,13 +96,30 @@ public function bake($name)
      */
     public function bakeTemplate($name)
     {
-        $templatePath = implode(DS, ['Template', 'Cell', $name, 'display.ctp']);
         $restore = $this->pathFragment;
-        $this->pathFragment = $templatePath;
 
+        $this->pathFragment = 'Template/Cell/';
         $path = $this->getPath();
+        $path .= implode(DS, [$name, 'display.ctp']);
+
         $this->pathFragment = $restore;
 
         $this->createFile($path, '');
     }
+
+    /**
+     * Gets the option parser instance and configures it.
+     *
+     * @return \Cake\Console\ConsoleOptionParser
+     */
+    public function getOptionParser()
+    {
+        $parser = parent::getOptionParser();
+        $parser
+        ->addOption('prefix', [
+            'help' => 'The namespace/routing prefix to use.'
```

Comment:
Is `prefix` intended to be connected to a routing prefixes here?

Reply:
It is not, I copied from different class. Fixed